src/db.c

```diff
@@ -2399,11 +2400,17 @@ int bitfieldGetKeys(struct redisCommand *cmd, robj **argv, int argc, getKeysResu
         int remargs = argc - i - 1; /* Remaining args other than current. */
         char *arg = argv[i]->ptr;
         if (!strcasecmp(arg, "get") && remargs >= 2) {
-            keys[0].flags = CMD_KEY_RO | CMD_KEY_ACCESS;
-            return 1;
+            readonly = 1;
+        } else if ((!strcasecmp(arg, "set") || !strcasecmp(arg, "incrby")) && remargs >= 3) {
```

Comment:
Good catch! 
Maybe we should use this opportunity to improve the arg parsing to actually understand the command syntax? 
I.E. When running into an operand, increment `i` to skip the known args.

Reply:
skipping the known args is ok. but I think we need not to check the syntax error, WDYT?

Reply:
Yes, we can assume they'll be rejected later on.. Just need to avoid crashing.

Reply:
updated